docs/interactions/api.rst

```diff
@@ -336,7 +336,7 @@ Enumerations
         Represents a link button.
     .. attribute:: premium
 
-        Represents a gradient button denoting that buying a SKU is
+        Represents a blurple button denoting that buying a SKU is
```

Comment:
Why does this need to mention a colour at all? It could simply be omitted.

Reply:
```suggestion
        Represents a button denoting that buying a SKU is
```